apps/emqx_bridge_kafka/test/emqx_bridge_v2_kafka_consumer_SUITE.erl

```diff
@@ -351,3 +351,28 @@ t_bad_bootstrap_host(Config) ->
         )
     ),
     ok.
+
+t_custom_group_id(Config) ->
```

Comment:
consumer?

Reply:
The intention was "custom" indeed, since the default behavior is to generate one automatically.